content/blog/gitops-schedule-kubecon-na-2024/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: GitOps at KubeCon NA (Salt Lake City) 2024
+date: "2024-11-07"
+description: "GitOps Schedule at KubeCon Salt Lake City"
+author: Christian Hernandez
+thumbnail: kubecon_saltlake.jpeg
+---
+
+We know tracking all the things at KubeCon can be a challenge, so we  we gathered up all things related to GitOps that are happening at KubeCon NA in Salt Lake City in one blog post (hope it helps). Please let us know if we missed anything, and be sure to add these to your Schedule!
+
+## Tuesday, 12 November 2024 (Day 0)
+
+- [ArgoCon NA](https://sched.co/1iVjy)
+- [Accelerating Application Delivery with OpenTofu Controller and GitOps](https://sched.co/1jCbL) - Lucas Duarte & Tiago Reichert (AWS) at OpenTofu Day, Community Insights colo event
+- [Flux lightning talk: What’s Flux and What’s New?](https://sched.co/1iW8J) | Project Lightning Talk with Leigh Capili (Flox) & Tamao Nakahara (helix.ml)
+
+## Wednesday 13 November 2024 (Day 1)
+
+- [TUF: Secure Distribution Beyond Software - Marina Moore, Independent](https://sched.co/1hovM)
+- [GitOops... I Did It Again! Protecting Your GitOps System from Being Used for Privilege Escalation - Oreen Livni & Elad Pticha, Cycode](https://sched.co/1i7nd)
+- [Towards Zero Change Incidents: Intuit's Strategy for Implementing AI-Driven Progressive Delivery - Avik Basu & Saravanan Balasubramanian, Intuit](https://sched.co/1i7kp)
+- [Mastering ApplicationSet: Advanced Argo CD Automation - Alexander Matyushentsev, Akuity](https://sched.co/1hovt)
+- [Perform Laser Focused Deployments by Deciding in Advance the Blast Radius - Kostis Kapelonis, Octopus deploy](https://sched.co/1i7mH)
+- [Taming Your Application’s Environments - Marcos Lilljedahl, Dagger & Mauricio "Salaboy" Salatino, Diagrid](https://sched.co/1i7kY)
+- [Poster Session (PS10): Accepting Mortality: Strategies for Ultra-Long Running Stateful Workloads in K8s - Sebastian Beyvers & Maria Hansen, Giessen University](https://sched.co/1i7mo)
+
+## Thursday 14 November 2024 (Day 2)
+
+- [GitOps at Production Scale with Flux - Leigh Capili, Flox & Priyanka Ravi, G-Research](https://sched.co/1howJ)
```

Comment:
```suggestion
- [GitOps at Production Scale with Flux - Leigh Capili, Flux & Priyanka Ravi, G-Research](https://sched.co/1howJ)
```

Reply:
Leigh is actually at a company called Flox! https://flox.dev/